package.json

```diff
@@ -5,10 +5,12 @@
     "url": "git://github.com/datasciencebr/jarbas.git"
   },
   "scripts": {
-    "postinstall": "elm-package install --yes"
+    "postinstall": "elm-package install --yes",
+    "test": "./node_modules/.bin/elm-test jarbas/frontend/tests/elm-tests/Main.elm"
```

Comment:
No need for `./node_modules/.bin/` here. Commands executed through NodeJS always look into `./node_modules/.bin/` by default ; )

In other words, it could be just `elm-test jarbas/frontend/tests/elm-tests/Main.elm`.